README.md

```diff
@@ -51,7 +51,7 @@ spec:
     fullSnapshotSchedule: 0 */24 * * *
     garbageCollectionPeriod: 43200s
     garbageCollectionPolicy: Exponential
-    imageRepository: eu.gcr.io/gardener-project/gardener/etcdbrctl
+    imageRepository: europe-docker.pkg.dev/gardener-project/public/gardener/etcdbrctl
```

Comment:
Have the older etcdbrctl images been copied over to artifact registry? Atleast `europe-docker.pkg.dev/gardener-project/public/gardener/etcdbrctl:v0.25.0` can't be pulled. Will artifact registry be used only for newer releases which explicitly push to it?

Reply:
@shreyas-s-rao : yes, I copied `europe-docker.pkg.dev/gardener-project/public/gardener/etcdbrctl:v0.25.0` from GCR -> AR prior to having opened this pull-request (I just double-checked: the images does exist and can be retrieved e.g. using docker).

So for special-case of v0.25.0 there should not be an issue. For future versions, new images will be published into AR.